bin/runtime/src/lib.rs

```diff
@@ -678,9 +678,9 @@ impl pallet_contracts::Config for Runtime {
     type DeletionQueueDepth = DeletionQueueDepth;
     type DeletionWeightLimit = DeletionWeightLimit;
     type Schedule = Schedule;
-    type CallStack = [pallet_contracts::Frame<Self>; 31];
+    type CallStack = [pallet_contracts::Frame<Self>; 5];
```

Comment:
Just FYI - this may cause currently deployed contracts to stop working.

Reply:
yes, but if your contract needs 5 nested cross-contract calls, then probably you don't have an optimal architecture anyway

Reply:
Yes. But previously you didn't know that when dry-running so could have decided to deploy it. I'm not blocking, just saying we should remember to mention it in the release notes.

Reply:
I'm kind of concerned why this is so small. Note that in EVM this is 256, so even 31 was already small, but now we are changing it to 5?! Would be great to know the core reason here...

Reply:
5 seems too small to me, each `delegate_call`-type upgradable contract already adds 2 frames. This is severely limiting the smart contract functionality